### ISSUE TITLE
Test error "Error: A sequence of more than one item is not allowed as the first argument of contains()"

### ISSUE THREAD

**Issue by vinnakota** [opengeospatial/ets-wms11#69]
I have couple of tests failing because of this error `Error: A sequence of more than one item is not allowed as the first argument of contains() ("application/vnd.esri.wms_raw_x...", "nosniff")`

Test Ids:
* wmsops-getfeatureinfo-params-query_layers-1
* wmsops-getfeatureinfo-params-query_layers-2

For each of these tests the server response headers contains content-type and x-content-type-options keys.
```
cache-control: private
content-encoding: gzip
content-length: 430
content-type: application/vnd.esri.wms_raw_xml
date: Thu, 13 Dec 2018 18:12:57 GMT
expires: Thu, 01 Jan 1970 00:00:00 UTC
server: Microsoft-IIS/10.0
server: 
status: 200
x-aspnet-version: 4.0.30319
x-content-type-options: nosniff
x-powered-by: ASP.NET
x-xss-protection: 1; mode=block
```

@ssankaran

**Comment by dstenger** [comment]
Thank you for reporting.

We will do further investigation.

**Comment by keshav-nangare** [comment]
@vinnakota 

Can you please provide the service URL?

**Comment by vinnakota** [comment]
@keshav-nangare https://ogccert.dev.geocloud.com/server/services/cite_shp/MapServer/WMSServer?request=GetCapabilities&service=WMS&version=1.1.1

**Comment by keshav-nangare** [comment]
Fixed with this commit: 17488df3ad746f19675e21c2a060c89a05813679
